src/org/loklak/api/search/SearchServlet.java

```diff
@@ -357,7 +360,7 @@ public void run() {
             }
             String rss = feed.toString();
             //System.out.println("feed has " + feed.size() + " entries");
-            
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.